estate/models/estate_property.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+
+from odoo import api, fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+    name = fields.Char(string="Title")
```

Comment:
We usually split here (I think it is in the guidelines)
```suggestion
    _description = "Estate Property"

    name = fields.Char(string="Title")
```